lib/erblint-github/linters/github/accessibility/navigation_has_label.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require_relative "../../custom_helpers"
+
+module ERBLint
+  module Linters
+    module GitHub
+      module Accessibility
+        class NavigationHasLabel < Linter
+          include ERBLint::Linters::CustomHelpers
+          include LinterRegistry
+
+          MESSAGE = "The navigation landmark should have a unique accessible name via `aria-label` or `aria-labelledby`."
```

Comment:
Should we say/ check that users do not include "navigaiton" in the name

Reply:
I added it to the message. As for it being a check, I'm open to following up on it separately, maybe as a distinct rule.